### PR TITLE
LSR-1034 Support multi select and pages for new forms (Android)

### PR DESCRIPTION
**Pull request by nhpatt**

### CODE REVIEW

File: android/library/src/main/java/com/liferay/mobile/screens/ddl/model/DDMStructure.java [hunk 1]
```diff
@@ -107,9 +116,16 @@ public void setLocale(Locale locale) {
...
+		this.name = jsonObject.getString("nameCurrentValue");
+		this.structureKey = jsonObject.getString("structureKey");
+		this.structureId = jsonObject.getString("structureId");
+		this.classNameId = jsonObject.getLong("classNameId");
+		this.classPK = "com.liferay.dynamic.data.mapping.model.DDMStructure";
 		if (jsonObject.has("xsd")) {
 			parse(jsonObject.getString("xsd"), new XSDParser());
-		} else {
+		}
+		else {
```
Comment (nhpatt) [inline android/library/src/main/java/com/liferay/mobile/screens/ddl/model/DDMStructure.java]: Fixed

File: android/library/src/main/java/com/liferay/mobile/screens/ddl/model/DDMStructure.java [hunk 2]
```diff
@@ -119,8 +135,9 @@ protected void parse(String content, DDMStructureParser parser) {
 		try {
 			Locale locale = this.locale == null ? LiferayLocale.getDefaultLocale() : this.locale;
 			fields = parser.parse(content, locale);
-		} catch (Exception e) {
+		}
+		catch (Exception e) {
```
Comment (nhpatt) [inline android/library/src/main/java/com/liferay/mobile/screens/ddl/model/DDMStructure.java]: Fixed

File: ios/Framework/Core/Asset/DisplayScreenlet/AssetDisplayScreenlet.swift [hunk 3]
```diff
@@ -15,7 +15,8 @@ import UIKit
 
 /// The AssetDisplayScreenletDelegate protocol defines some methods that you use to manage the
 /// AssetDisplayScreenlet events. All of them are optional.
-@objc public protocol AssetDisplayScreenletDelegate: BaseScreenletDelegate {
+@objc(AssetDisplayScreenletDelegate)
```
Comment (nhpatt) [inline ios/Framework/Core/Asset/DisplayScreenlet/AssetDisplayScreenlet.swift]: But if the PR doesn't need a rebase/merge and can be integrated directly this is not a problem. I haven't touched this file and develop will have the latest version.

File: android/samples/test-app/src/main/res/values/server_context.xml [hunk 4]
```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
-	<string name="liferay_server">http://screens-demo.liferay.org.es</string>
+	<string name="liferay_server">http://screens.liferay.org.es</string>
```
Comment (nhpatt) [inline android/samples/test-app/src/main/res/values/server_context.xml]: Exactly the same as the previous comment. I haven't deleted the commit, will pick the commit when github does the merge

File: ios/Framework/Core/Asset/DisplayScreenlet/AssetDisplayScreenlet.swift (hunk 3 above)
Comment (dgarciasarai) [inline ios/Framework/Core/Asset/DisplayScreenlet/AssetDisplayScreenlet.swift]: That's true ;)

**Comment by nhpatt** [comment]
I've added some new commits @dgarciasarai

File: android/viewsets/material/src/main/java/com/liferay/mobile/screens/viewsets/material/ddl/form/ExtendedViewPagerDDLFormView.java [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+
+	@Override
+	public void showFailedOperation(String actionName, Exception e, Object argument) {
+		if (actionName.equals(DDLFormScreenlet.LOAD_FORM_ACTION)) {
+			LiferayLogger.e("error loading DDLForm", e);
+			clearFormFields();
+		}
+	}
+
+//	@Override
```
Comment (dgarciasarai) [inline android/viewsets/material/src/main/java/com/liferay/mobile/screens/viewsets/material/ddl/form/ExtendedViewPagerDDLFormView.java]: Why is this commented?

**Comment by dgarciasarai** [comment]
LGTM 👍 Please, update CHANGELOG file and merge :)

**Pull request merged**